topi/include/topi/elemwise.h

```diff
@@ -169,5 +171,124 @@ inline Tensor cast(const Tensor& x,
   }, name, tag);
 }
 
+/*!
+* \brief Creates an operation that sum each element of a tensor
+*
+* \param xs The input tensor array
+* \param name The name of the operation
+* \param tag The tag to mark the operation
+*
+* \return A Tensor whose op member is the sum operation
+*/
+inline Tensor elemwise_sum(const Array<Tensor>& xs,
+                           std::string name = "tensor",
+                           std::string tag = kElementWise) {
+  CHECK_GT(xs.size(), 0) << "elemwise sum must have at least one input tensor.";
+  return compute(xs[0]->shape, [&](const Array<Var>& i) {
+      auto sum_expr = xs[0](i);
+      for (size_t j = 1; j < xs.size(); j++) {
+        sum_expr = sum_expr + xs[j](i);
+      }
+      return sum_expr;
+  }, name, tag);
+}
+
+/*!
+* \brief Creates an operation that fill a tensor with fill_value
+*
+* \param shape The shape of a tensor
+* \param dtype The number of arguments
+* \param fill_value The value to be filled
+* \param name The name of the operation
+* \param tag The tag to mark the operation
+*
+* \return A Tensor filled with fill_value
+*/
+inline Tensor full(const Array<Expr>& shape,
```

Comment:
why is the name full. It's operation is to fill the tensor with value fill_value. Shouldn't the name be 'fill'?
@tqchen python op also has the name 'full'. Can you clarify here.

Reply:
mxnet and numpy use `full` as operation name, but tensorflow use `fill`. maybe it's a matter of taste.  I'll follow community's direction. 

* https://docs.scipy.org/doc/numpy-1.14.0/reference/generated/numpy.full.html
* https://mxnet.incubator.apache.org/api/python/symbol/symbol.html#mxnet.symbol.full
* https://www.tensorflow.org/api_docs/python/tf/fill

Reply:
ok